docs/02_getting-started/03_smart-contract-development/04_data-persistence.md

```diff
@@ -92,14 +92,13 @@ struct person {
 Now that the data structure of the table has been defined with a `struct` we need to configure the table. The [eosio::multi_index](https://developers.eos.io/manuals/eosio.cdt/latest/classeosio_1_1multi__index) constructor needs to be named and configured to use the struct we previously defined.
 
 ```cpp
-typedef eosio::multi_index<"people"_n, person> address_index;
+using address_index = eosio::multi_index<"people"_n, person>;
 ```
 With the above `multi_index` configuration there is a table named **people**, that
 
 1. Uses the `_n` operator to define an eosio::name type and uses that to name the table. This table contains a number of different singular "persons", so name the table "people".
 2. Pass in the singular `person` struct defined in the previous step.
 3. Declare this table's type. This type will be used to instantiate this table later.
-4. There are some additional configurations, such as configuring indices, that will be covered further on.
```

Comment:
is this line deleted by mistake?

Reply:
No, it was intentional. It is a bit talk about future which I think it should be removed